static/js/settings_exports.js

```diff
@@ -22,6 +22,16 @@ exports.populate_exports_table = function (exports) {
         name: "admin_exports_list",
         modifier: function (data) {
             if (data.export_data.deleted_timestamp === undefined) {
+                var export_status = $('#export_status');
+                if (export_status.hasClass('alert-success')) {
+                    // Politely remove our success banner if the export
+                    // finishes before the view is closed.
+                    export_status.fadeTo(200, 0);
+                    setTimeout(function () {
+                        export_status.hide();
+                    }, 205);
+                }
+
```

Comment:
I would have expected this change to involve modifying the `server_events_dispatch` code path to just call a little function that removes the banner?  I don't see the benefit of having this add complexity to the `list_render` code.

Reply:
Ahh gotcha, I misunderstood what you were implying in [this](https://github.com/zulip/zulip/issues/13045#issuecomment-523699274) comment. I'll update to use this approach.